app/frontend/src/features/auth/AuthPage.tsx

```diff
@@ -47,61 +62,145 @@ const useStyles = makeStyles((theme) => ({
     height: "50vh",
     justifyContent: "center",
     textAlign: "center",
+    [theme.breakpoints.up("md")]: {
+      height: "100%",
+      margin: "0 auto",
+      width: "960px",
```

Comment:
`theme.breakpoints.values.md` (or is it `lg`)?

Reply:
It is md, changed! https://material-ui.com/customization/breakpoints/